### PR TITLE
Use PVC instead of hostpath for EOS automounts

### PR DESCRIPTION
**Pull request by PMax5**

### CODE REVIEW

File: swan/files/swan_config.py [hunk 1]
```diff
@@ -175,21 +175,20 @@ def swan_pod_hook(spawner, pod):
...
             name='eos',
             mount_path='/eos',
             mount_propagation='HostToContainer'
-        ),
-    )
-    c.SwanKubeSpawner.volumes.append(
-        V1Volume(
-            name='eos',
-            host_path=V1HostPathVolumeSource(
-                path=get_config("custom.eos.automountHostPath", "/var/eos")
```
Comment (etejedor) [inline swan/files/swan_config.py:190]: `custom.eos.automountHostPath` is not needed anymore, then? We can perhaps get rid of the anchor and just set it as a config argument for the eosxd-csi chart?

File: swan/templates/eos/automount-pvc.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
```
Comment (etejedor) [inline swan/templates/eos/automount-pvc.yaml:1]: We can call the file just `pvc.yaml`?

File: swan/files/swan_config.py (hunk 1 above)
Comment (PMax5) [reply swan/files/swan_config.py:190]: I removed it yes

File: swan/templates/eos/automount-pvc.yaml (hunk 2 above)
Comment (PMax5) [reply swan/templates/eos/automount-pvc.yaml]: That is also true, changed.

**Review by etejedor — APPROVED** [review]

**Pull request merged**